python/cudf/cudf/pylibcudf_tests/test_lists.py

```diff
@@ -1,21 +1,26 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
+import numpy as np
 import pyarrow as pa
 import pytest
 from utils import assert_column_eq
 
+import cudf
```

Comment:
pylibcudf tests shouldn't be using cudf. I see usage of some cudf DeviceScalar objects, we should be only using pylibcudf Scalars here.